pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -1,12 +1,11 @@
 #
 # Progression of infection within individuals
 #
-
 import random
-
+import pyEpiabm as pe
 from pyEpiabm.property import InfectionStatus
-
 from .abstract_sweep import AbstractSweep
+from pyEpiabm.utility import InverseCdf
```

Comment:
Please order and space as per PEP8 guidelines - I would suggestion a correction but can't because of the number of deleted lines. Other files have examples, but imports should be grouped by origin (ie standard library/third party/local) in that order, with a blank line between each group.